haystack/document_stores/sql.py

```diff
@@ -20,24 +20,22 @@
 class ORMBase(Base):
     __abstract__ = True
 
-    id = Column(String(100), default=lambda: str(uuid4()), primary_key=True)
+    id = Column(String(100), default=lambda: str(uuid4()), unique=True, primary_key=True)
     created_at = Column(DateTime, server_default=func.now())
     updated_at = Column(DateTime, server_default=func.now(), server_onupdate=func.now())
 
 
 class DocumentORM(ORMBase):
     __tablename__ = "document"
 
+    # Composite PK with id allows the same doc in many indices
```

Comment:
How about separating this and creating id to index mapping in another table?

Reply:
I think id to index will have a one-to-many relationship

Reply:
This is actually not a bad idea, but I'm not sure it will solve the problem. I'll keep it in mind though, thanks!

Reply:
>  I'm not sure it will solve the problem

Then, this table will not have multiple PKs